### ISSUE TITLE
[MCU] CAN driver

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#10]
## User story
1. As a developer
2. I want / need to send can messages without dealing with the hardware
3. So that the code I write stays portable

## Acceptance criteria
* .h-file without platform specific includes defines functions for sending can message and registering callback for specific ID
* .h-file for platform has initialization method
* .c-file implements initialization (except initializing peripheral, which must be done before) and functionality

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
